docs/standard/containerized-lifecycle-architecture/road-to-modern-applications-based-on-containers.md

```diff
@@ -1,19 +1,19 @@
 ---
 title: Road to modern applications based on containers
-description: Understand that Docker, containers and a Microservices architecture are not a one-size-fits-all solution. Here you will find some references that will help you decide.
+description: Understand that Docker, containers and a Microservices architecture are not a one-size-fits-all solution. Here you'll find some references that will help you decide.
 author: CESARDELATORRE
 ms.author: wiwagn
 ms.date: 11/23/2018
 ---
 # Road to modern applications based on containers
 
-You are probably reading this book because you are planning the development of new applications and/or you are assessing the impact of using Docker, Containers, and new approaches like Microservices in your company.
+You're probably reading this book because you're planning the development of new applications and/or you're assessing the impact of using Docker, Containers, and new approaches like Microservices in your company.
```

Comment:
```suggestion
You're probably reading this book because you're planning the development of new applications or you're assessing the impact of using Docker, Containers, and new approaches like Microservices in your company.
```